client/src/legacy/LeftAndMain.EditForm.js

```diff
@@ -607,15 +607,6 @@ $.entwine('ss', function($){
         this.hideMembersList(instant);
       }
     },
-    // these two functions retained for backwards compatibility
-    showList: function(instant) {
-      this.showGroupsList(instant);
-      this.showMembersList(instant);
-    },
-    hideList: function(instant) {
-      this.hideGroupsList(instant);
-      this.hideMembersList(instant);
-    },
```

Comment:
These were only kept in case someone was calling them directly in their own code. See https://github.com/silverstripe/silverstripe-admin/pull/1531#pullrequestreview-1513487040
Now that we're doing a major release, we can remove these unnecessary functions.